latex/interni/doc_interna/norme_di_progetto/res/sections/01_content.tex

```diff
@@ -118,10 +118,33 @@ \subsection{Verifica}
 Dopo aver eseguito un commit sul ramo da integrare approva la Pull Request e conferma il merge secondo le norme di progetto descritte nella sezione dedicata.
 
 \subsection{Approvazione}
-IN SOSPESO
+L'approvazione viene svolta dal Responsabile di Progetto.\\
+Si effettua questa pratica non appena il file prodotto viene verificato e passa tutte le eventuali reviews create dai verificatori e non necessita di ulteriori modifiche o revisioni future. Spetterà quindi l'ultima parola al Responsabile di Progetto che dovrà approvare la versione del file. In caso si riscontrassero ulteriori problematiche, verrà riportata la pull request nella fase di verifica citata precedentemente nel punto 2.2. \\
+Ad un esito positivo di questa fase di approvazione, verrà aggiunto il nome di chi ha approvato il file nell'intestazione prodotta dal file titlepage\_input.tex, creando una nuova riga nel registro delle modifiche, nel file changelog\_input.tex, inserendo l'ultima versione secondo le regole di versionamento e scrivendo nelle rispettive colonne il suo nome e la descrizione "Approvazione".\\
+Verrà poi chiusa la pull request da parte del Responsabile di Progetto e l'issue associata di approvazione del file, come spiegato dalla sezione dedicata, si chiuderà automaticamente.
+
 \subsection{Strumenti per la stesura}
 \begin{itemize} 
-    \item Latex: IN SOSPESO
+    \item LaTeX: è un linguaggio di marcatura per la preparazione di testi, basato sul 				  		  programma di composizione tipografica TEX.\\
+		  I nostri documenti sono scritti utilizzando solamente questo linguaggio. Nel branch 				  documentation  si possono trovare i file .pdf prodotti e la cartella “latex” che 					  conterrà a sua volta tre cartelle interne:
+	\begin{itemize}
+	\item esterni e interni, contengono file .tex di documentazione esterna ed interna come ad esempio i verbali o altra documentazione esterna/interna
+	\begin{itemize}
+		\item Ogni documento prodotto contiene:
+		\begin{itemize}
+		\item la cartella config, dove troviamo i file di template .tex riscritti in base al documento che si sta producendo
+		\item la cartella res/sections, dove troviamo i file .tex delle sezioni del documento
+		\item un file col nome del documento pdf con estensione .tex in cui raggruppa tutti gli altri file .tex prodotti nelle due cartelle sopracitate
+		\end{itemize}
+	\end{itemize}
+	 \item template, contiene file .tex di base utilizzati in ogni documento prodotto
```

Comment:
ci sono dei documenti che non contengono tutte le parti (esempio tracciamento temi affrontati viene messo solo nei verbali)

Reply:
template, contiene i file .tex di base utilizzati secondo necessità per comporre i documenti.

nota:In questo modo restiamo generici